rest_framework/fields.py

```diff
@@ -56,10 +56,18 @@ def is_simple_callable(obj):
     if not (function or method):
         return False
 
-    args, _, _, defaults = inspect.getargspec(obj)
-    len_args = len(args) if function else len(args) - 1
-    len_defaults = len(defaults) if defaults else 0
-    return len_args <= len_defaults
+    if six.PY2:
```

Comment:
We put all version branching like this in `compat.py`. If you switch against <= 3.2 instead then we'd be able to accept this. I'd assume we'll drop 3.2 support on the next major release in any case.